### ISSUE TITLE
MessageBox: [A11y Issue]The title and contents should be read out by screen reader

### ISSUE THREAD

**Issue by herlinsong** [SAP/ui5-webcomponents-react#1477]
**Describe the bug**
When the messageBox is opened, the title and contents in the MessageBox should be read out by screen reader automatically.

**Isolated Example**
UI5-WC-React Demo:  
https://fme37.csb.app/

UI5 Demo:
https://sapui5.hana.ondemand.com/#/entity/sap.m.MessageBox/sample/sap.m.sample.MessageBox

**To Reproduce**
1. Turn on screen reader and open the UI5-WC-React Demo
2. Click on the button 'Open MessageBox', the screen reader will not read the title and the content.
3. Open the UI5 Demo
4. Click on any button to open the MessageBox, the screen reader will read the title and the content automatically.

**Expected behavior**
In the UI5-WC-React Demo, the title and contents in the MessageBox should be read out by screen reader automatically when opened.
Expect the same behavior like https://sapui5.hana.ondemand.com/#/entity/sap.m.MessageBox/sample/sap.m.sample.MessageBox

**Screenshots**
![image](https://user-images.githubusercontent.com/81735931/113256401-b9685080-92fb-11eb-823c-4fc4cb497a4a.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: Current Version
`@ui5/webcomponents-react` version: Current Version
Operating System: Windows
Browser: Chrome

**Additional context**
I am from SAP SuccessFactors

**Comment by Lukas742** [comment]
released in [0.15.3](https://github.com/SAP/ui5-webcomponents-react/releases/tag/v0.15.3)
